contracts/modules/Liquidation.sol

```diff
@@ -7,7 +7,11 @@ import "../BaseLogic.sol";
 
 /// @notice Liquidate users who are in collateral violation to protect lenders
 contract Liquidation is BaseLogic {
-    constructor(bytes32 moduleGitCommit_) BaseLogic(MODULEID__LIQUIDATION, moduleGitCommit_) {}
+    constructor(bytes32 moduleGitCommit_, uint32 selfCollateralFactor_) BaseLogic(MODULEID__LIQUIDATION, moduleGitCommit_) {
+        selfCollateralFactor = selfCollateralFactor_;
```

Comment:
I don't like it either. Note that it would be like this even if we left it as a constant in Constants.sol. Both modules could become out of sync if they were compiled at different times (different git revisions), or only one was deployed, etc.

Maybe we could 100% contain it in RiskManager by making Liquidation do an internal module call to RM?

Reply:
Hard to say. A call is a bit of extra gas that translates to a tiny bit more of bad debt, and like you say, its not much different from constants. It is one extra thing to be mindful though during deployments. Maybe we could verify it in the deployment script? Check the constructor arg matches for Liquidation and RiskManager?